replicatingtopics/src/main.py

```diff
@@ -0,0 +1,14 @@
+from replicator import Replicator
+
+from kafka import KafkaConsumer, KafkaProducer
+
+
+def main(replicator: Replicator):
+    replicator.replicate('a', 'b')
```

Comment:
try read topics from config file and avoid hard coding in code for topic names

Reply:
done

Reply:
but i am not going to close  this pr since complete review isn't complete you will find the config file in the next pr .